recipes/piqp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "piqp" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PREDICT-EPFL/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9af1fef8308782bd7df2d7aef551cba60d42e983f485e258c9ed23a523cded0c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - vs2022_win-64  # [win]
+    - python
+    - eigen
+    - setuptools >=42
+    - wheel
+    - make  # [not win]
+    - ninja
+    - cmake >=3.12
+    - pip
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - scipy
+    - numpy
```

Comment:
scipy and numpy aren't part of the install_requires. Are you sure you want to make them run dependencies?